### ISSUE TITLE
Multiple Packet per Connection Interval

### ISSUE THREAD

**Issue by StanislasGaitUp** [NordicSemiconductor/Android-BLE-Library#68]
I am working on an application using your library and a home made Nordic sensor 
On some devices (Samsung S7, Galaxy Tab A) I am seeing the same problem as here : https://stackoverflow.com/questions/24817107/android-receiving-multiple-ble-packets-per-connection-interval

When the sensor sends N packets (p1, p2 ... pN) in the same connexion interval  the Library calls the callback "Received PN" N times

However it works perfectly on other Android Devices and all iOS test devices I have.

**Comment by philips77** [comment]
Hi, I replied on the StackOverflow topic.
Do you still see the problem in 2.0.3?

**Comment by StanislasGaitUp** [comment]
Hi, thank you very much for your fast answer,
Actually, I am sorry the problem is not exactly the same as the stack overflow, I am using the Notification system and not the onCharacteristicChanged. But the result is the same : when I get many quick notifications, the values get overriden to the last one.
I updated the value to 2.0.3 and the problem seems to have been fixed. I will let you know if this problem comes back
